src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -220,35 +235,69 @@ class OppgaveService(
 
             log.info("Mapper funnet: Antall: ${mapperResponse.antallTreffTotalt}, ${mapperResponse.mapper} ")
+            val toggleEnabled =
+                featureToggleService.isEnabled("familie.ef.mottak.mappe.selvstendig.tilsynskrevende")
 
             val mappe =
-                if (oppgave.behandlingstema == BEHANDLINGSTEMA_SKOLEPENGER && oppgave.tildeltEnhetsnr == ENHETSNUMMER_NAY) {
-                    hentOpplæringsmappeSkolepenger(mapperResponse)
+                if (erSkolepenger(oppgave)) {
+                    finnMappe(mapperResponse, "65 Opplæring")
+                } else if (toggleEnabled && harTilsynskrevendeBarn(søknadId, oppgave)) {
+                    finnMappe(mapperResponse, søkestreng = "60 Særlig tilsynskrevende")
+                } else if (toggleEnabled && erSelvstendig(søknadId, oppgave)) {
+                    finnMappe(mapperResponse, søkestreng = "61 Selvstendig næringsdrivende")
                 } else {
-                    hentMappeEfSakUpplassert(mapperResponse)
+                    finnMappe(mapperResponse, "01 Uplassert")
                 }
-
             integrasjonerClient.oppdaterOppgave(oppgaveId, oppgave.copy(mappeId = mappe.id.toLong()))
         } else {
             secureLogger.info("Flytter ikke oppgave til mappe $oppgave")
         }
     }
 
-    private fun hentMappeEfSakUpplassert(mapperResponse: FinnMappeResponseDto) =
-        (
-            mapperResponse.mapper.find { it.navn.contains("EF Sak", true) && it.navn.contains("01") }
-                ?: error("Fant ikke mappe for uplassert oppgave (EF Sak og 01)")
-            )
+    private fun erSkolepenger(oppgave: Oppgave) =
+        oppgave.behandlingstema == BEHANDLINGSTEMA_SKOLEPENGER && oppgave.tildeltEnhetsnr == ENHETSNUMMER_NAY
+
+    private fun erSelvstendig(søknadId: String?, oppgave: Oppgave) =
+        if (søknadId != null && oppgave.behandlingstema == BEHANDLINGSTEMA_OVERGANGSSTØNAD) {
+            val søknadJson = søknadService.get(søknadId).søknadJson
+            val søknadOvergangsstønad = objectMapper.readValue<SøknadOvergangsstønad>(søknadJson.data)
+            erSelvstendig(søknadOvergangsstønad.aktivitet)
+        } else if (søknadId != null && oppgave.behandlingstema == BEHANDLINGSTEMA_BARNETILSYN) {
+            val søknadJson = søknadService.get(søknadId).søknadJson
+            val søknadBarnetilsyn = objectMapper.readValue<SøknadBarnetilsyn>(søknadJson.data)
+            erSelvstendig(søknadBarnetilsyn.aktivitet)
+        } else {
+            false
+        }
 
-    private fun hentOpplæringsmappeSkolepenger(
-        mapperResponse: FinnMappeResponseDto,
-    ) = (
-        mapperResponse.mapper.find {
-            it.navn.contains("EF Sak", true) &&
-                it.navn.contains("65 Opplæring", true)
+    private fun erSelvstendig(aktivitet: Søknadsfelt<Aktivitet>) =
+        aktivitet.verdi.firmaer?.verdi?.isNotEmpty() ?: false ||
+            aktivitet.verdi.virksomhet?.verdi != null
+
```

Comment:
Har dere vurdert å gjøre denne "sjekken" på `familie-ef-søknad-api`-siden og sende med 
```
{ 
  ... søknadsdata,
  spesialhåndtering: ER_SELVSTENDIG | TILSYNSKREVENDe_BARN | INGEN,
}
```

rett til mottak?

Isåfall slipper mottak å parse søknadsinnholdet og være oppdatert på "siste skrik" :)